pkg/odo/cli/component/push.go

```diff
@@ -102,7 +103,18 @@ func (po *PushOptions) Validate() (err error) {
 
 // Run has the logic to perform the required actions as part of command
 func (po *PushOptions) Run() (err error) {
-	return po.Push()
+	err = po.Push()
+	if err != nil {
+		return err
+	}
+	if log.IsJSON() {
+		componentDesc, err := component.GetComponent(po.Context.Client, po.Context.Component(), po.Context.Application, po.Context.Project)
```

Comment:
Not sure if outputting component description is what we want for `odo push -o json` output. 
I would skip JSON output for the push for now as we are not sure how it should look like. This PR was supposed to just about create command.

Reply:
Sorry, skipped